### PR TITLE
Update ruby to 3.2.2 and postgres image to 15

### PR DESCRIPTION
**Pull request by guillermoap**
- Update ruby version to 3.2.2 both in docker and in .ruby-version file.
- Update postgres image version in docker-compose to 15

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
-ruby '~> 3.1.2'
+ruby '~> 3.2.2'
```
Comment (santib) [inline Gemfile:4]: I think some gems will break. For example [rspec_api_documentation](https://github.com/zipmark/rspec_api_documentation/pull/507)

Comment (guillermoap) [reply Gemfile:4]: we should probably remove that gem as well since it's pretty outdated.

Comment (santib) [reply Gemfile:4]: we need to find a replacement, not just removing it

Comment (guillermoap) [reply Gemfile:4]: Will create a new pr removing that and exception_hunter, updating pagy and some other stuff I found.

Comment (guillermoap) [reply Gemfile:4]: rswag seems like the closest thing to it

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.1.2
+FROM ruby:3.2.2
```
Comment (JulianPasquale) [inline Dockerfile:1]: Can we update the `.dockerignore` file here as well? 🙏

Comment (santib) [reply Dockerfile:1]: maybe this should be done in a separate PR. I don't think this one will be merged for quite some time

**Comment by santib** [comment]
When running tests we get a warning
`To use retry middleware with Faraday v2.0+, install `faraday-retry` gem`

**Review by santib — APPROVED** [review]
🤩

**Review by sebastiancaraballo — APPROVED** [review]

**Review by jpascual1994 — APPROVED** [review]

**Pull request merged**